sdk/src/Services/S3/Custom/AmazonS3Client.Extensions.cs

```diff
@@ -46,6 +46,23 @@ namespace Amazon.S3
 {
     public partial class AmazonS3Client : AmazonServiceClient, IAmazonS3
     {
+        /// <summary>
+        /// A hardcoded list for regions that support SigV2 for S3 endpoints to preserve legacy behavior. 
+        /// New regions shouldn't support SigV2 as it is a deprecated signature version. This is a copy
+        /// of the same private member in RegionEndpoint.
```

Comment:
The only reference to `_sigV2SupportedRegions` in the `RegionEndpoint` was removed, can we delete the field from that class? (and update this comment since it won't be a copy anymore)